doc/docs/master/concepts/advanced-concepts/deferred_processing.md

```diff
@@ -112,17 +113,22 @@ To configure a staging branch, complete the following steps:
    The `master` and `staging` branches now have the same `HEAD` commit.
    This means that your pipeline has data to process.
 
+!!! note
+   In some rare cases, such as if a pipeline is directly provenant on both `master` and
+   an input to `master`, the commit IDs may differ. Deferred processing will still occur
+   correctly in this case, but the new commits won't be in the old commit set.
+
```

Comment:
This note might be too obscure to matter

Reply:
Removed the note for now - waiting for a new `pachctl trace commit` command to add a similar explanation back. See  #6555